packages/vsphere-cpi/charts/values.yaml

```diff
@@ -0,0 +1,11 @@
+vCenter:
+  host: ""
+  insecureflag: "1"
+  port: "443"
```

Comment:
This is alarming, but looks like its only used in conf files.

Reply:
Yes, this is only for the vsphere's cloud-config file.